src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet
+{ 
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
```

Comment:
I understand what we are trying to achieve here (thanks for the reminder :smile:). However, putting this information in the SdkVersion field violates the [single responsibility principle](http://www.butunclebob.com/ArticleS.UncleBob.PrinciplesOfOod) of OO and the [first normal form](http://en.wikipedia.org/wiki/First_normal_form) of Database design, which leads to known consequences. Will the data pipeline separate the "source" information into a different field in storage schema for filtering and segmentation? Are we working with them to introduce a new field for this information in the data collection schema?